ParallaxView.podspec

```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
     s.summary = "Add parallax effect like in tvOS applications to any view."
     s.requires_arc = true
     s.license = "MIT"
-    s.version = "2.0.8"
+    s.version = "5.0.0"
```

Comment:
Why did you set `version` to 5.0.0? Probably you mix it up with [swift_version](https://guides.cocoapods.org/syntax/podspec.html#swift_versions).

Reply:
fixed it